src/pseudopeople/entity_types.py

```diff
@@ -63,8 +63,8 @@ def __call__(
         to_noise_idx = get_index_to_noise(
             column, noise_level, randomness_stream, f"{self.name}_{additional_key}"
         )
-        column.loc[to_noise_idx] = self.noise_function(
+        noised_data = self.noise_function(
             column.loc[to_noise_idx], configuration, randomness_stream, additional_key
         )
-
+        column.loc[to_noise_idx] = noised_data
```

Comment:
Was this actually causing a problem or do you just find this more readable?

Reply:
This just made debugging easier since I could put a breakpoint between the function call and the assignment to the series.